src/resources/js/v2/tickets-loader.js

```diff
@@ -0,0 +1,79 @@
+/* global tribe */
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.tickets = tribe.tickets || {};
+
+/**
+ * Configures ET Loader Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.tickets.loader = {};
+
+/**
+ * Initializes in a Strict env the code that manages the plugin "loader".
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.tickets.loader
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	const $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup.
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		loader: '.tribe-common-c-loader',
+		hiddenElement: '.tribe-common-a11y-hidden',
+	};
+
+	/**
+	 * Show loader for the container.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of the container.
+	 *
+	 * @return {void}
+	 */
+	obj.show = function( $container ) {
+		const $loader = $container.find( obj.selectors.loader );
+
+		if ( $loader.length ) {
+			$loader.removeClass( obj.selectors.hiddenElement.className() );
```

Comment:
Sorry, not familiar with the `.className()` on the `obj.selectors.hiddenElement` property, it seems like a string though and you don't need to use `className()`?

Reply:
It's a prototype function from `tribe-common.js` that we use to be sure we're using the class name as we need it: https://github.com/moderntribe/tribe-common/blob/master/src/resources/js/tribe-common.js#L2-L15